src/Components/Articles.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export default function Articles() {
+  const [articleList, setArticleLsit] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-6m81.onrender.com/api/articles")
+      .then((response) => {
+        setArticleLsit(response.data);
+      });
+  }, []);
+  return (
+    <div>
+      <h1>Articles</h1>
+      <ul className="article-container">
+        {articleList.map((article) => {
+          return (
+            <li className="article-card">
+              <p>{article.votes} 👍</p>
+              {article.title}
```

Comment:
I'd position the votes and author at the bottom of the image, as this would be the general design for a news app 👍